temporal/temporalintro.html

```diff
@@ -160,6 +160,24 @@ <h4>Querying and map calculation</h4>
     <li><a href="t.rast.neighbors.html">t.rast.neighbors</a></li>
 </ul>
 
+<p>
+Similar to the <em>where</em> option, selected modules like
+<a href="t.rast.univar.html">t.rast.univar</a> support a
+<em>spatial_relation</em> option to limit computations to maps of
+the space time raster dataset that have a given spatial relation to the
+current computational region. Supported spatial relations are:
+<ul>
+ <li>"overlaps": process only maps that spatially overlap ("intersect")
+with the current computational region</li>
+ <li>"is_contained": process only maps that are fully within the current
+computational region</li>
+ <li>"contains": process only maps that contain (fully cover) the current
+computational region</li>
+</ul>
+Space time raster datasets may contain raster maps with varying spatial
+extent like for example series of scenes of satellite images. For such
+cases the <em>spatial_relation</em> option can be useful.
```

Comment:
Now this should say region_relation, no?

Reply:
Same for the module doc.

Reply:
Addressed in [593da28](https://github.com/OSGeo/grass/pull/2793/commits/593da2826502023b56be96197b994ae394f7018a) and [403072a](https://github.com/OSGeo/grass/pull/2793/commits/403072a2adeb318c4ad1c3a0b288736bd73bb644)